### PR TITLE
Run licensecheck

### PR DESCRIPTION
**Pull request by akurtakov**

### CODE REVIEW

**Comment by akurtakov** [comment]
Licensecheck fails to start with https://github.com/eclipse-cbi/org.eclipse.cbi/actions/runs/8230419948 . @mbarbero @fredg02 should this one be allowed? I would say yes as dash-licenses uses it by default.

**Comment by fredg02** [comment]
@akurtakov AFAICT there has been no HelpDesk issue requesting the setup of the license vetting workflow for the CBI project, so far. Can you open one?

**Comment by akurtakov** [comment]
Here you are https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/4416

File: .github/workflows/licensecheck.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  issue_comment:
+    types: [created]
+
+jobs:
+  call-license-check:
+    uses: eclipse/dash-licenses/.github/workflows/mavenLicenseCheck.yml@master
+    with:
+      projectId: technology.cbi
+    secrets:
+      gitlabAPIToken: ${{ secrets.CBI_GITLAB_API_TOKEN }}
```
Comment (fredg02) [inline .github/workflows/licensecheck.yml:21]: Please use `${{ secrets.ECLIPSE_GITLAB_API_TOKEN }}`.

Comment (akurtakov) [reply .github/workflows/licensecheck.yml]: Done

**Review by fredg02 — APPROVED** [review]

**Comment by akurtakov** [comment]
@fredg02 It still doesn't run licensecheck as can be seen at https://github.com/eclipse-cbi/org.eclipse.cbi/actions/runs/8237852312 . There must be some org or repo config allowing only given set of GH actions.

**Comment by akurtakov** [comment]
E.g. smth like https://docs.github.com/en/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization but I'm allowed to see these configs.

**Comment by netomi** [comment]
can be enabled via self-service.

**Comment by netomi** [comment]
running octopin I get the following list of actions:

```
/home/tn/workspace/eclipse/eclipse-cbi/org.eclipse.cbi/.github/workflows/licensecheck.yml
└── eclipse/dash-licenses/.github/workflows/mavenLicenseCheck.yml@master
    ├── actions/cache@v4
    ├── actions/checkout@v4
    ├── actions/checkout@v4
    ├── actions/github-script@v7
    ├── actions/github-script@v7
    ├── actions/setup-java@v4
    ├── actions/upload-artifact@v4
    ├── eclipse/dash-licenses/.github/actions/maven-license-check-action@master
    └── stCarolas/setup-maven@v5

```

**Comment by netomi** [comment]
test

**Comment by netomi** [comment]
Created https://github.com/eclipse-cbi/.eclipsefdn/pull/11

**Comment by netomi** [comment]
should work now.

**Comment by fredg02** [comment]
> E.g. smth like https://docs.github.com/en/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization but I'm allowed to see these configs.

With the self-service you can see almost all the configs:
https://github.com/eclipse-cbi/.eclipsefdn/blob/main/otterdog/eclipse-cbi.jsonnet

**Comment by fredg02** [comment]
@akurtakov please test if it works as expected now.

**Comment by akurtakov** [comment]
/request-license-review on https://github.com/eclipse-cbi/org.eclipse.cbi/pull/425 did nothing so I guess I screwed smth.

**Pull request merged**